src/fmu/dataio/dataio.py

```diff
@@ -290,8 +294,10 @@ class ExportData:
             [[20200101, "monitor"], [20180101, "base"]] or just [[2021010]]. The output
             to metadata will from version 0.9 be different (API change)
 
-        vertical_domain: This is dictionary with a key and a reference e.g.
-            {"depth": "msl"} which is default if missing.
+        vertical_domain: Optional. String with vertical domain either "time" or "depth".
```

Comment:
I see we use "Optional ", "Optional. ", and "Optional, " to indicate optionality. It's not relevant to this PR, just an observation, but maybe we should align all of these to one if it's reasonable to do so.

Reply:
yes, I agree the entire argument documentation should have a go through